experimental/BasisLieHighestWeight/src/TensorModels.jl

```diff
@@ -30,9 +31,9 @@ function tensor_matrices_of_operators(
     end
     wi = ZZ.(1:length(highest_weight) .== i) # i-th fundamental weight
     matrices_of_operators = [
-      _tensor_product(mat_temp, _tensor_power(mat_wi, highest_weight_i)) for
+      _tensor_product(mat_temp, _tensor_power(mat_wi, 1)) for
       (mat_temp, mat_wi) in
-      zip(matrices_of_operators, matrices_of_operators_gap(L, wi, operators))
+      zip(matrices_of_operators, matrices_of_operators_gap(L, highest_weight[i]*wi, operators))
```

Comment:
```suggestion
      zip(matrices_of_operators, matrices_of_operators_gap(L, highest_weight_i*wi, operators))
```
bereits existierende variable nutzen